CONP_main_data_documentation.htm

```diff
@@ -0,0 +1,27 @@
+
+
+<h1>Uploading data for CONP</h1>
+
+<h2>1. Metadata</h2>
+
+We use the DATS.json file format to store metadata about CONP datasets. DATS.json is a flexible machine-readable structure allowing for sophisticated representation of a wide range of different types of datasets, from neuroimaging to genomics.  A DATS.json file must be prepared for each dataset.
+
+The CONP DATS.json schema includes the following required fields:
```

Comment:
the excel file also has the identifier, types and version as required fields.

Reply:
Is the excel file now an authoritative final version ?

Reply:
I see it the other way around. Necessity comes from usage. Those fields are nice to have but not providing them wont break anything. Not having a title could.

Reply:
Jen was talking yesterday about making a nice neat table in a Word document from that spreadsheet once JB signed off on the decisions Jen and I made updating the spreadsheet yesterday morning, and I am expecting to replace the excel file that is currently there, and also update the list in the main documentation page, when I receive that table.